cfme/tests/cli/test_appliance_console.py

```diff
@@ -52,6 +53,31 @@
 ]
 
 
+def is_port_open(hostname, port):
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.connect((hostname, int(port)))
+        logger.info("Port %s is open on %s host" % (hostname, port))
+        return True
+    except Exception:
+        logger.info("Port %s is not open on %s host" % (hostname, port))
+        return False
+
+
+def wait_for_web_ui_up(app_obj, old_uptime):
+    """
+    input:
+        app_obj: appliance object
+        old_uptime: last uptime of appliance (appliance.ssh_client.uptime())
+    output:
+        None
+    """
+    client = app_obj.ssh_client
+    wait_for(lambda: client.uptime() < old_uptime, handle_exception=True,
+             num_sec=600, delay=2)
+    app_obj.wait_for_web_ui()
```

Comment:
I don't think what you're doing here requires a different function. I think you should just includes this in the test function below, unless you plan on using this function in a different test case.